webknossos-tracingstore/app/com/scalableminds/webknossos/tracingstore/tracings/volume/VolumeTracingService.scala

```diff
@@ -95,7 +95,7 @@ class VolumeTracingService @Inject()(
               case a: UpdateBucketVolumeAction =>
                 if (tracing.getMappingIsEditable) {
                   Fox.failure("Cannot mutate volume data in annotation with editable mapping.")
-                } else updateBucket(tracingId, tracing, a, updateGroup.version)
+                } else updateBucket(tracingId, tracing, a, updateGroup.version) ?~> "Failed to save volume data."
```

Comment:
When I provoked the error, I only saw this message here and not the "received a mag...". is this intentional? should there be an error chain?

Reply:
I think there should be a chain. I’ll have another look

Reply:
You were right, there was a conversion that ate the error chain. Should be fixed now :)